docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -1278,3 +1278,85 @@ function su_humsci_profile_update_9724() {
 
   $config->set('allowed_entity_types.node', $allowed_entity_types)->save();
 }
+
+/**
+ * Clean-up possible duplicates from su_humsci_profile_update_9723().
+ */
+function su_humsci_profile_update_9725(&$sandbox) {
+  $node_type = 'hs_private_page';
+  $results = [];
+  // Remove any dupes from bundle settings.
+  $settings = content_access_get_settings('all', $node_type);
+  foreach ($settings as $operation => &$permissions) {
+    if (!is_array($permissions)) {
+      // This isn't actually an array of perms, this is another setting.
+      continue;
+    }
+    $cleanPermissions = array_unique($permissions);
+    if ($cleanPermissions !== $permissions) {
+      $permissions = $cleanPermissions;
+      $results[] = t('Cleaned %operation.', ['%operation' => $operation]);
+    }
+  }
+  // Without enforcing per-node settings we got lots of dupes in the
+  // node_access table due to a clash between the manual items set in
+  // su_humsci_profile_update_9723(), plus the automatic entries that Content
+  // Access was trying to add (when per-node was not enabled).  Those dupes
+  // caused fatal exceptions.
+  if (empty($settings['per_node'])) {
+    $settings['per_node'] = TRUE;
+    $results[] = t('Enforced per-node Content Access.');
+  }
+  if ($results) {
+    content_access_set_settings($settings, $node_type);
+    $results[] = t('Saved Content Access settings for Private Pages.');
+    node_access_rebuild(TRUE);
+    $results[] = t('Rebuilt node access.');
+    return implode('  <br />', $results);
+  }
+}
+
+/**
+ * Clean-up possible duplicates from su_humsci_profile_update_9723().
+ */
+function su_humsci_profile_update_9726(&$sandbox) {
```

Comment:
I'm not sure if this second is needed but it also doesn't appear to hurt anything.